master-thesis.md

```diff
@@ -0,0 +1,53 @@
+# Master Thesis Topics in Project Chains
+
+1. [Reproducible Builds for non-compiled languages like JavaScript](#reproducible-builds-for-non-compiled-languages-like-javascript)
```

Comment:
can we have an automated outline in Github pages?

Reply:
![image](https://github.com/user-attachments/assets/a7188eaf-2d36-47af-a294-b527ab6ff473)

Outline:
![image](https://github.com/user-attachments/assets/25764c7b-9788-4a8b-847c-d4747bd97aeb)

GitHub has this feature natively (without Refined GitHub). Do you think this is sufficient? Otherwise we will have to go for a CI script which updates this page.